cluster-bootstrap/router-default-svc-config.yml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: router
+    router: router-default
+  name: router-default
+  namespace: openshift-ingress
+spec:
+  externalTrafficPolicy: Local
+  ports:
+  - name: http
+    nodePort: ${ROUTER_NODEPORT_HTTP}
+    port: 80
+    protocol: TCP
+    targetPort: http
+  - name: https
+    nodePort: ${ROUTER_NODEPORT_HTTPS}
+    port: 443
+    protocol: TCP
+    targetPort: https
+  selector:
+    ingresscontroller.operator.openshift.io/deployment-ingresscontroller: default
+  sessionAffinity: None
+  type: NodePort
```

Comment:
this seems to be a mirror of the `router-internal-default` svc the ingress-operator creates, only as type `NodePort`.

the router pods run in the host network namespace so... i'm not sure why this is needed.

Reply:
I need a service for which I can set the ports so I can expose them via the AWS load balancer. At the time I create the AWS infrastructure I don't know what ports will be assigned to router-internal-default.